### DISCUSSION TITLE
Would Capacitor V3 iOS be supported by both CoCoaPods and Carthage?

### DISCUSSION THREAD

**Discussion by buleglay** [ionic-team/capacitor#3649] [General]
I understand the current version of Capacitor for iOS is a dependency managed via CocoaPods.

For the new version, is there any discussion to make it available on Carthage as well?

That would give users the flexibility to choose the dep management system they like, for example, someone would like to install other plugins for the native part via Carthage.

**Comment by jcesarmobile** [comment]
Capacitor is really distributed through npm, it's not even published on CocoaPods anymore.
But it relies on CocoaPods for installing it as a local library, plugins are also distributed on npm but they are also local CocoaPods libraries. And Capacitor converts Cordova plugins to be local CocoaPod libraries.
I doubt we could make all that work as both CocoaPods and Carthage, specially the Cordova plugins part.

**Reply by buleglay** [reply]
Thanks for the detailed explanation.
